runtime/server/batch_query_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"testing"
 
 	runtimev1 "github.com/rilldata/rill/proto/gen/rill/runtime/v1"
+	_ "github.com/rilldata/rill/runtime/resolvers"
```

Comment:
Underscore imports should be in a separate group (linter enforces this for non-test files)